<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
		background-color:  qlineargradient(
		spread:pad, x1:0, y1:0 x2:0, y2:0,
		stop: 0 #A3D8F4, stop:1 #E6F7FF
		);
		border: 2px solid #ffffff;
		boder-radius: 10px;
}

QLineEdit {
		border: 2px solid #E63946;
		border-radius: 5px;
		padding: 5px;
		font-size: 14px;
		background-color: #ffffff;
		color: #333333;
}

QLineEdit:forcus {
		border: 2px solid #457B9D;
		background-color: #F1FAEE;
}

QPushButton{
		background-color: #2A9D8F;
		color: white;
		border: none;
		border-radius: 5px;
		padding: 10px 20px;
		font-size: 14px;
}

QPushButton:hover{
background-color: #264653;
}

QPushButton:pressed{
background-color: #1B3B3A;
}

QLabel{
font-size :14px;
color: #1D3557;
font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>835</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>400</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="removeRoomUserButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Members  </string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/group_chat.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inviteToRoomButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Add  </string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/plus.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="leaveRoomButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Leave  </string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/leave.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="styleSheet">
        <string notr="true">QDialog {
		background-color:  qlineargradient(
		spread:pad, x1:0, y1:0 x2:0, y2:0,
		stop: 0 #A3D8F4, stop:1 #E6F7FF
		);
		border: 2px solid #ffffff;
		boder-radius: 10px;
}

QLineEdit {
		border: 2px solid #E63946;
		border-radius: 5px;
		padding: 5px;
		font-size: 14px;
		background-color: #ffffff;
		color: #333333;
}

QLineEdit:forcus {
		border: 2px solid #457B9D;
		background-color: #F1FAEE;
}

QPushButton{
		background-color: #2A9D8F;
		color: white;
		border: none;
		padding: 10px 20px;
		font-size: 14px;
}

QPushButton:hover{
background-color: #264653;
}

QPushButton:pressed{
background-color: #1B3B3A;
}

QLabel{
font-size :14px;
color: #1D3557;
font-weight: bold;
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>831</width>
          <height>465</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="notificationsLayout"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>160</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="chatLayout"/>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="messageEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Send   </string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/send.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Friend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadButton">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/refresh_icon.png</normalon>
          </iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="onlineUsersList"/>
     </item>
     <item>
      <widget class="QPushButton" name="requestChat11">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Chat    </string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/chat.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="createRoomButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Create Room   </string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/add_group.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="friendRequestButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Add Friend   </string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/add_friend_icon.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="friendListButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Friend List   </string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/friend_list.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ChatRoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reloadButton_2">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/new/prefix1/refresh_icon.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="chatRoomsList"/>
     </item>
     <item>
      <widget class="QPushButton" name="joinRoomButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Join   </string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/join.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="removeRoomUserButton_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>171</width>
      <height>37</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>  User</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/new/prefix1/user.png</normalon>
     </iconset>
    </property>
   </widget>
   <widget class="QLabel" name="userName">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>6</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
